subsys/net/lib/app/net_app.c

```diff
@@ -506,13 +516,104 @@ struct net_context *select_client_ctx(struct net_app_ctx *ctx,
 #endif /* CONFIG_NET_APP_CLIENT */
 
 #if defined(CONFIG_NET_APP_SERVER)
+static struct net_context *get_server_ctx(struct net_app_ctx *ctx,
+					  const struct sockaddr *dst)
+{
+	int i;
+
+	for (i = 0; i < CONFIG_NET_APP_SERVER_NUM_CONN; i++) {
+		struct net_context *tmp;
+		u16_t port, rport;
+
+		if (!ctx->server.net_ctxs[i] ||
+		    !net_context_is_used(ctx->server.net_ctxs[i])) {
+			continue;
+		}
+
+		tmp = ctx->server.net_ctxs[i];
```

Comment:
Why is tmp assigned only here? Could do this in its definition and simplify "if" above.

Reply:
True, I will fix this.